app/jobs/scheduled/resend_validation_emails.rb

```diff
@@ -0,0 +1,22 @@
+module Jobs
+  class ResendValidationEmails < Jobs::Scheduled
```

Comment:
I would change the name of the job to be more relevant to what it's doing. Something like

```suggestion
  class ActivationReminderEmails < Jobs::Scheduled
```

Don't forget to also change the name of the file to match the name of the job.